src/Evervault/EvervaultHttp.php

```diff
@@ -4,31 +4,41 @@
 
 class EvervaultHttp {
     private $apiKey;
+    private $appUuid;
     private $apiBaseUrl;
     private $functionRunBaseUrl;
 
     private $curl;
 
     private $appKeyPath = '/cages/key';
     private $relayConfigPath = '/v2/relay-outbound';
+    private $decryptPath = '/decrypt';
     
     private $appKey;
 
-    function __construct($apiKey, $apiBaseUrl, $functionRunBaseUrl) {
+    function __construct($apiKey, $appUuid, $apiBaseUrl, $functionRunBaseUrl) {
         $this->apiKey = $apiKey;
+        $this->appUuid = $appUuid;
         $this->apiBaseUrl = $apiBaseUrl;
         $this->functionRunBaseUrl = $functionRunBaseUrl;
 
         $this->curl = curl_init();
     }
 
-    private function _getDefaultHeaders() {
-        return [
-            'api-key: '.$this->apiKey,
+    private function _getDefaultHeaders($basicAuth = false) {
+        $defaultHeaders = [
             'content-type: application/json',
             'accept: application/json',
-            'user-agent: evervault-php/'.Evervault::VERSION
+            'user-agent: evervault-php/'.Evervault::VERSION,
```

Comment:
```suggestion
            'user-agent: evervault-php/'.Evervault::VERSION
```

Trailing commas only came in in PHP 7.2, but we support >7.1